src/program/events.rs

```diff
@@ -65,6 +65,14 @@ pub struct FeeEvent {
     pub fees_collected_in_quote_lots: u64,
 }
 
+#[derive(Debug, Copy, Clone, BorshDeserialize, BorshSerialize)]
+pub struct TimeInForceEvent {
```

Comment:
do we want to include the current slot / timestamp here, or just get that from the chain?

Reply:
it's in the header

Reply:
word. timestamp too?